docs/en/latest/admin-api.md

```diff
@@ -94,7 +94,11 @@ Note: When the `Admin API` is enabled, it will occupy the API prefixed with `/ap
 | create_time      | False                                    | Auxiliary   | epoch timestamp in second, will be created automatically if missing                                                                                                                                                                                                                                                                                                                                                                | 1602883670                                           |
 | update_time      | False                                    | Auxiliary   | epoch timestamp in second, will be created automatically if missing                                                                                                                                                                                                                                                                                                                                                                | 1602883670                                           |
 
-For the same type of parameters, such as `host` and `hosts`, `remote_addr` and `remote_addrs` cannot exist at the same time, only one of them can be selected. If enabled at the same time, the API will respond with an error.
+There are three points that need special attention:
+
+- In addition to the required `uri`/`uris`, you must select at least one of the four categories `plugin`, `script`, `upstream`/`upstream_id` and `service_id`.
```

Comment:
> t least one of the four categories `plugin`, `script`, `upstream`/`upstream_id` and `service_id`

This is wrong.
They can be put into two groups:
plugin vs script
upstream vs upstream_id vs service_id

Reply:
Thanks for reminding, it has been updated.